src/chia_log/log_consumer.py

```diff
@@ -103,13 +105,33 @@ def stop(self):
         self._is_running = False
 
     def _consume_loop(self):
-        logging.info(f"Consuming remote log file {self._remote_log_path} from {self._remote_host}")
-        stdin, stdout, stderr = self._ssh_client.exec_command(f"tail -F {self._remote_log_path}")
+        logging.info(f"Consuming remote log file {self._remote_log_path} from {self._remote_host} ({self._remote_platform})")
+
+        if self._remote_platform == "windows":
+            stdin, stdout, stderr = self._ssh_client.exec_command(f"powershell.exe Get-Content {self._remote_log_path} -Wait -Tail 1")
+        else:
+            stdin, stdout, stderr = self._ssh_client.exec_command(f"tail -F {self._remote_log_path}")
 
         while self._is_running:
             log_line = stdout.readline()
             self._notify_subscribers(log_line)
 
+    def _get_remote_platform(self):
+        stdin, stdout, stderr = self._ssh_client.exec_command(f"uname -a")
+        fout: str = stdout.readline().lower()
+        ferr: str = stderr.readline().lower()
+
+        if 'linux' in fout:
+            return 'linux'
+        elif 'darwin' in fout:
+            return 'macos'
+        elif 'not recognized' in ferr:
+            return 'windows'
```

Comment:
Let's make `Enum` class for these platforms. Will be less error prone and can be reused in other places in the future. I would probably define it inside `util.py` file in `src` folder for now (but maybe you have better idea)

Reply:
Should be OK now!